### DISCUSSION TITLE
How to Binding input to Markdown with use_state

### DISCUSSION THREAD

**Discussion by bobwatcherx** [widgetti/solara#101] [Q&A] [answered]
how to binding value from inputText to markdown . i change textinput but markdown cannot change
my code
```
@solara.component
def MyHome():
    name,set_name = use_state("tolol")
    with AppBar():
        Title("tollll")
    with Column():
        Markdown(str(name))
        InputText("insert name",value=name,continuous_update=name)
```

**Comment by maartenbreddels** [comment, marked as answer]
If you pass on_value=set_name to inputtext your example will work. For simpler 2 way binding look into reactive variables:
  * https://solara.dev/api/reactive 
  * https://solara.dev/api/use_reactive

hope that makes sense! 

Regards,

Maarten
